server/protocols/webadmin-cli/src/test/java/org/apache/james/cli/DomainManageTest.java

```diff
@@ -121,4 +121,26 @@ void domainDeleteCommandWithDefaultDomainShouldFailed(GuiceJamesServer server) {
         assertThat(exitCode).isEqualTo(1);
     }
 
+    @Test
+    void domainExistCommandWithDefaultDomainShouldBeExisted(GuiceJamesServer server) {
+        Port port = server.getProbe(WebAdminGuiceProbe.class).getWebAdminPort();
+
+        int exitCode = WebAdminCli.executeFluent(new PrintStream(outputStreamCaptor), new PrintStream(errorStreamCaptor),
+            "--url", "http://127.0.0.1:" + port.getValue(), "domain", "exist", "localhost");
+
+        assertThat(exitCode).isEqualTo(0);
+        assertThat(outputStreamCaptor.toString().trim()).isEqualTo("localhost exists");
+    }
+
+    @Test
+    void domainExistCommandNotWithDefaultDomainShouldNotBeExisted(GuiceJamesServer server) {
+        Port port = server.getProbe(WebAdminGuiceProbe.class).getWebAdminPort();
+
+        int exitCode = WebAdminCli.executeFluent(new PrintStream(outputStreamCaptor), new PrintStream(errorStreamCaptor),
+            "--url", "http://127.0.0.1:" + port.getValue(), "domain", "exist", "linagora.com");
+
+        assertThat(exitCode).isEqualTo(0);
+        assertThat(outputStreamCaptor.toString().trim()).isEqualTo("linagora.com does not exist");
+    }
```

Comment:
How about adding a test when you add a new domain, then check it exists?

Reply:
Solved